stats/view_test.go

```diff
@@ -594,6 +584,298 @@ func Test_View_MeasureFloat64_AggregationCount_WindowSlidingTime(t *testing.T) {
 	}
 }
 
+func Test_View_MeasureFloat64_AggregationSum_WindowCumulative(t *testing.T) {
+	k1, _ := tag.NewKey("k1")
+	k2, _ := tag.NewKey("k2")
+	k3, _ := tag.NewKey("k3")
+	agg1 := SumAggregation{}
```

Comment:
s/agg1/agg

Reply:
Do we need this as a variable, what about:

view, err := NewView("VF1", "desc VF1", []tag.Key{k1, k2}, nil, SumAggregation{}, Cumulative{})